src/auth/auth.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
+import * as argon2 from 'argon2';
+import { JwtPayload, Tokens } from 'src/@types/auth';
+
+@Injectable()
+export class AuthService {
+  constructor(private jwtService: JwtService) {}
+
+  login(payload: JwtPayload): Tokens {
+    const accessToken = this.jwtService.sign(payload, {
+      expiresIn: '2h',
```

Comment:
Think it would be good to talk about the expiration time

Reply:
Good!